libmpv/client.h

```diff
@@ -26,6 +26,14 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#if __GNUC__ || __clang__
+#define EXPORT __attribute__((visibility("default")))
+#elif _MSC_VER
+#define EXPORT __declspec(dllexport)
+#else
+#define EXPORT
```

Comment:
should be `MPV_EXPORT` to avoid conflicts with who knows what

Also consider adding a comment that new symbols must still *also* be added to `libmpv/mpv.def`